plugins/crust-gather.yaml

```diff
@@ -0,0 +1,48 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: crust-gather
+spec:
+  version: v0.5.3
+  homepage: https://github.com/crust-gather/crust-gather
+  platforms:
+  - bin: kubectl-crust-gather.exe
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_windows_amd64.tar.gz
+    sha256: 6c3e3697ad33ad190fd2398776fc9a2b28288147c57991878c9d48ed38063f23
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_linux_arm64.tar.gz
+    sha256: e4c01ab0b26af5771aba07b29ab303976841aace58b71605d816267f377d6df6
+    selector:
+      matchLabels:
+        os: linux
+        arch: arm64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_linux_amd64.tar.gz
+    sha256: 61b8dc87d4cabccc628fce55359b30d14b2d2363667d17696b98ba1f825f8582
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_darwin_arm64.tar.gz
+    sha256: 506fb5be53170cb2f9edb0842b616aed8e76c8b832f61d9e46ec2f636d880500
+    selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+  - bin: kubectl-crust-gather
+    uri: https://github.com/crust-gather/crust-gather/releases/download/v0.5.3/kubectl-crust-gather_v0.5.3_darwin_amd64.tar.gz
+    sha256: 506fb5be53170cb2f9edb0842b616aed8e76c8b832f61d9e46ec2f636d880500
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Provides functionality similar to os adm inspect, by collecting all resources for debugging purposes, and serving them via simple api-server.
+  description: |
+    Provides functionality similar to os adm inspect, by collecting all resources for debugging purposes. Later, the archive can be served
+    with an api-server to explore the content using regular clients, like kubectl or k9s.
```

Comment:
I wouldn't mention "os adm inspect" here. Forget about that and approach writing this from this perspective:  I'm someone who's looking at this tool for the first time, what does it do for me. Focus on that.


Also consider line wrapping this at 80 chars please.

Reply:
Updated. Makes sense, the description is very much outdated, as the plugin works on a different pattern.